sql/src/test/java/org/opensearch/sql/sql/parser/QueryDataAnonymizerTest.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+
+package org.opensearch.sql.sql.parser;
+
+import org.antlr.v4.runtime.tree.ParseTree;
+import org.junit.jupiter.api.Test;
+import org.opensearch.sql.sql.antlr.SQLSyntaxParser;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class QueryDataAnonymizerTest {
+    SQLSyntaxParser parser = new SQLSyntaxParser();
```

Comment:
take a look at the doctest files for good 'high-level' tests.  
e.g. we should have: `highlight`, a `match`, `position(`, `match_bool_prefix` as tests
include LIMIT, HAVING, ORDER BY

Reply:
I have added tests involving all of those functions as well as LIMIT, HAVING, and ORDER BY in https://github.com/Bit-Quill/opensearch-project-sql/pull/266/commits/4151c87037d711794af02cb86bb9c9e7c228f94c